styles.css

```diff
@@ -1,44 +1,61 @@
 * {
+    text-align: center;
+    box-sizing: border-box;
+}
+
+body {
     margin: 0 auto;
-    outline: 1px
+    width: 960px;
 }
 
 @media only screen and (max-width: 768px) and (min-width: 480px)  {
```

Comment:
notice that when you shrink your screen all the way down that nothing shows up at all! That's because of these conditions

Reply:
Yikes! That's no good...I'm on it!

Reply:
Ah ha! My breakpoints were whacky! Should be fixed now